stripe/src/main/java/com/stripe/android/PaymentController.java

```diff
@@ -58,7 +58,9 @@
 
     PaymentController(@NonNull Context context,
                       @NonNull StripeApiHandler apiHandler) {
-        this(context, new StripeThreeDs2ServiceImpl(context), apiHandler,
+        this(context,
```

Comment:
feels like an odd indentation choice just for the first param haha, but if CI is ok w/ it, I am too :)

Reply:
😁